src/components/download-item/download-item.tsx

```diff
@@ -76,7 +75,7 @@ export const DownloadItem = withText({
             data-testid="download-item-download-button"
             className={styles.fileInfo}
             tabIndex={0}
-            aria-label={downloadButtonLabel}
+            aria-label={`${downloadButtonLabel} ${fileName}${description ? ' ' + description : ''}`}
```

Comment:
Much simpler - 
${[fileName, description].join(" ")}

Reply:
the issue with this that it add redundant space at the end in case it is undefined 
`aria-label="Click to download the attachment Q4 Slides  "`

but it is ignored by the screen reader so i guss it is not critical

Reply:
without a space in between ?